dev/TreeView/TreeViewItem.xaml

```diff
@@ -118,244 +121,11 @@
                             </Grid.ColumnDefinitions>
                             
                             <Grid Grid.Column="0">
-                                <Grid.Resources>
```

Comment:
Just so I'm sure I understand: TreeView is just using the stock check box now?  Why did it previously have its own check box style?

Reply:
Yeah that was by design, TreeView had its own CheckBox style, if you look closely at the CheckBox in TreeView today, the background and stroke colors are different from the default CheckBox.

But now we do want a consistent style so I switched to the stock CheckBox.

Reply:
Gotcha.  Thanks!